### PR TITLE
Created experiment doc for speed of sound experiment

### PR DESCRIPTION
**Pull request by abhinavraj23**
Fixes issue #675 

Changes: Created the .md file experiment document for speed of sound experiment document

 Screenshots for the change: 
speedofsound

### CODE REVIEW

**Comment by abhinavraj23** [comment]
[app-debug.zip](https://github.com/fossasia/pslab-android/files/1750507/app-debug.zip)

File: app/src/main/assets/DOC_HTML/apps/SpeedOfSound.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Speed of sound
+---
+	
+![](file:///android_asset/DOC_HTML/apps/images/schematics/HCSR04.svg@100%|auto)	
+  
+* Make the Connections as shown in the figure
+* The Ultrasonic Range Finder HC-SR04 measures the distance of an obstacle from the sensor 
+* It first emits a high frequency sound (40KHz) , and starts a stopwatch. This emitted sound pulse reflects off the nearest object, and returns to the sensor,
```
Comment (CloudyPadmal) [inline app/src/main/assets/DOC_HTML/apps/SpeedOfSound.md]: 40KHz --> 40 kHz

Comment (CloudyPadmal) [inline app/src/main/assets/DOC_HTML/apps/SpeedOfSound.md]: and starts a stopwatch --> and starts a timer

File: app/src/main/assets/DOC_HTML/apps/SpeedOfSound.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+   which immediately stops the stopwatch.
```
Comment (CloudyPadmal) [inline app/src/main/assets/DOC_HTML/apps/SpeedOfSound.md]: stops the timer.

**Comment by abhinavraj23** [comment]
Thank you @CloudyPadmal, till yesterday the code was being approved by travis-ci but now it is not, I don't know what happened??

**Comment by abhinavraj23** [comment]
Thank you @asitava1998

**Pull request merged**